sprinklers/base.py

```diff
@@ -29,6 +42,8 @@ class SprinklerBase(object):
     # Task limit per chord, no limit by default
     chord_size = None
```

Comment:
@colin-rhodes maybe worth documenting that this parameter is (I think) ignored if sharding is on.

Reply:
And by "documenting" i just mean "leaving a comment in code"

Reply:
yeah planning on removing chord size. It's used in the codebase on one sprinkler and rather than removing it outright I figured it's easier to implement the parallel path. I don't think we need chord_size at all if we have sharded sprinklers